libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -99,3 +129,28 @@ object MapboxMetricsReporter : MetricsReporter {
         this.metricsObserver = null
     }
 }
+
+/**
+ * Read url and token from resources if they are available
```

Comment:
Is this documented somewhere? I mean which keys the user should put in order to override token and url? Or is it an internal feature?